src/django/api/management/commands/post_deployment.py

```diff
@@ -9,3 +9,4 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         call_command('migrate')
+        call_command('index_facilities_new')
```

Comment:
Are you going to keep `index_facilities_new` on post deploy forever, or this is only for this PR?